<launch>
  <arg name="port" default="/dev/ttyUSB2"/>
  <param name="port" value="$(arg port)"/>
  <node name="gps_publisher" pkg="gps_driver" type="driver.py" output="screen">
  </node>
</launch>
